src/style.scss

```diff
@@ -321,9 +321,15 @@
   }
 }
 
+:host([showOverflow]) {
+  .details {
+    overflow: visible;
+  }
+}
+
```

Comment:
I am not excited about adding a new API that does basically the same thing. I'd argue that we remove the `noanimation` attribute from the JSDocs and deprecate that feature.